docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: '2.4'
 services:
 
   nginx:
-    image: "grocy/nginx:v3.0.0-1"
+    image: "grocy/nginx:${GROCY_IMAGE_TAG:-latest}"
     build:
       args:
         GROCY_VERSION: v3.0.0
```

Comment:
@BitFis Any thoughts on whether we could/should automate this value too?

Reply:
How do you want to automate that?

Reply:
It'd be tricky; we'd probably have to parse the git tag and remove the suffix, so that `tag:v3.0.0-1` -> `GROCY_VERSION:v3.0.0` - and that could be fragile.

I think I'll merge this as-is, check that it's working, and we can think about further automation in future.